cfme/infrastructure/provider/__init__.py

```diff
@@ -167,7 +167,7 @@ def vm_name(self):
         return version.pick({
             version.LOWEST: "VMs",
             '5.5': "VMs and Instances",
-            '5.7': "Virtual Machines"})
+            '5.7.1': "Virtual Machines"})
```

Comment:
Well, if you started doing that everywhere else, I will do the same here. Thanks for info